### ISSUE TITLE
psnr

### ISSUE THREAD

**Issue by small-White-JJ** [gangsterless/pytorch-softmax-splatting#9]
请问有测试psnr和ssim吗？
